project/modules/cache0.sc

```diff
@@ -59,4 +59,6 @@ trait CacheJvmBase extends Cache with CsMima with Shading {
       super.sources() ++ Seq(PathRef(dest))
     }
   }
+  @deprecated("Use CacheJvmBaseTests")
+  trait Tests extends CacheJvmBaseTests
```

Comment:
Could we remove it altogether? It's only used in the coursier build, no?

Reply:
Yeah, I needed it in-between to let the compiler give me more precise messages. I'll remove later.